crits/samples/templates/samples_detail.html

```diff
@@ -75,6 +76,16 @@ <h3 class='titleheader'>
                 </td>
             </tr>
             <tr>
+                <td class="key">Filepaths</td>
```

Comment:
I noticed you have a `filepath` attribute as well as a `filepaths` attribute (I assume for the same reason we did this for filename..so there's one main filepath you want to refer to, with references to other places it could be). I don't, however, see the `filepath` exposed in the details template?